sdk/fileapi_sdk/Common/extended_property.h

```diff
@@ -0,0 +1,17 @@
+﻿#ifndef API_IMPL_EXTENDED_PROPERTY_H_
+#define API_IMPL_EXTENDED_PROPERTY_H_
+
+#include <string>
+
+namespace mip {
+
+struct ExtendedProperty
+{
+  std::string vendor;

```

Comment:
according to Munjal - spaces

---
In reply to: [134187360](https://github.com/AzureAD/rms-sdk-for-cpp/pull/132#discussion_r134187360) [](ancestors = 134187360)